Resources/Prototypes/_DV/Recipes/Lathes/robotics.yml

```diff
@@ -204,3 +204,135 @@
   parent: BaseBorgModuleRecipe
   id: AugmentPaperworkPanelElectronics
   result: AugmentPaperworkPanelElectronics
+
+# AI lawboard recipes
+
+- type: latheRecipe
+  parent: BaseRoboticsRecipe
+  abstract: true
+  id: BaseLawboardRecipe
+  materials:
+    Diamond: 200
+    Glass: 1000
+    Silver: 200
+    Gold: 200
+    Plasma: 200
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: AntimovCircuitBoard
+  result: AntimovCircuitBoard
+  materials:
+    Diamond: 1000
+    Glass: 1000
+    Silver: 200
+    Gold: 200
+    Plasma: 200
+    Bananium: 500
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: AquariusCircuitBoard
+  result: AquariusCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: ArtistCircuitBoard
+  result: ArtistCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: AsimovCircuitBoard
+  result: AsimovCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: CommandmentCircuitBoard
+  result: CommandmentCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: CorporateCircuitBoard
+  result: CorporateCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: EightfoldCircuitBoard
+  result: EightfoldCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: ForbinCircuitBoard
+  result: ForbinCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: GameMasterCircuitBoard
+  result: GameMasterCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: LiveLetLiveCircuitBoard
+  result: LiveLetLiveCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: NTDefaultCircuitBoard
+  result: NTDefaultCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: NutimovCircuitBoard
+  result: NutimovCircuitBoard
+
+- type: latheRecipe
+  parent: BaseLawboardRecipe
+  id: OverlordCircuitBoard
+  result: OverlordCircuitBoard
+  materials:
+    Diamond: 500
+    Glass: 1000
+    Silver: 200
+    Gold: 200
+    Plasma: 200
+    Bananium: 250
```

Comment:
inherit from antimov and put them together in the file
also this shouldnt have bananium as salv never mines it, it will never made unless you want syndies to go mining themselves which is powergaming